griptape/drivers/memory/conversation/dynamodb_conversation_memory_driver.py

```diff
@@ -0,0 +1,52 @@
+from attr import define, field
+from boto3 import resource
+from griptape.drivers import BaseConversationMemoryDriver
+from griptape.memory.structure import ConversationMemory
+
+
+@define
+class DynamoDbConversationMemoryDriver(BaseConversationMemoryDriver):
+    aws_region: str = field(default=None, kw_only=True)
+    table_name: str = field(default=None, kw_only=True)
+    partition_key: str = field(default=None, kw_only=True)
+    value_attribute_key: str = field(default=None, kw_only=True)
+    partition_key_value: str = field(default=None, kw_only=True)
```

Comment:
Sounds like of those should be required, right? If so, we should remove the defaults.